### ISSUE TITLE
Install instructions

### ISSUE THREAD

**Issue by PhrozenByte** [nextcloud/integration_whiteboard#4]
Couldn't find any info about how to actually install this app. It requires a separate installation of https://github.com/spacedeck/spacedeck-open, right? Spacedeck requires a Node.js server, so basically one must run Nextcloud on a VPS since least shared webhosters support Node.js?

**Comment by socialize-IT** [comment]
No. You don't need a seperate installation of spacedeck. Just make shure the needed libraries are installed as stated here: https://github.com/nextcloud/integration_whiteboard#-install

After installing and activating the app you can use the whiteboard.

**Comment by szaimen** [comment]
*at least as long as you can install the app ;)
https://github.com/nextcloud/integration_whiteboard/issues/3

**Comment by socialize-IT** [comment]
Yes. Don't know what the problem is, but installing manually is working.

**Comment by eneiluj** [comment]
The app does not require NodeJS or any external Spacedeck instance deployed. Spacedeck is bundled (with Nexe) in the app, automatically configured and launched by Nextcloud.

graphicsmagick, ffmpeg and ghostscript are optional requirements. It will works without them installed but you will miss some Spacedeck features.

v0.0.13 is out. This release fixes the archive extraction issue.

**Comment by sudwhiwdh** [comment]
> graphicsmagick, ffmpeg and ghostscript are optional requirements. It will works without them installed but you will miss some Spacedeck features.

Does it make sense to include a check in the integration if graphicsmagick, ffmpeg and ghostscript are already present in the system and if necessary offer to install them or at least point out which features won't work without them?

**Comment by ei8fdb** [comment]
> Does it make sense to include a check in the integration if graphicsmagick, ffmpeg and ghostscript are already present in the system and if necessary offer to install them or at least point out which features won't work without them?

I think yes. I spent a large chunk of time today cursing Spacedeck and Nextcloud thinking "why are my images not displaying. damn thing!".

I came to the repo to report an issue, and then saw that graphicsmagick, ffmpeg and ghostscript are required. 

It would also reduce the number of issues being opened incorrectly.
